apps/dialtone-documentation/docs/components/emoji-picker.md

```diff
@@ -76,21 +76,115 @@ storybook: https://dialtone.dialpad.com/vue/vue3/?path=/story/components-emoji-p
+              <svg>...</svg>
+            </span>
+            <input name="" type="text" class="base-input__input d-input d-input-icon--left" data-qa="dt-input-input" id="searchInput" placeholder="Search..." />
+          </div>
+        </label>
+      </div>
+    </div>
+    <div class="d-emoji-picker__selector">
+      <div id="d-emoji-picker-list" class="d-emoji-picker__list">
+        <div class="d-emoji-picker__category d-emoji-picker__alignment" data-index="0"><p>Most recently used</p></div>
+        <div class="d-emoji-picker__alignment" data-index="1">
+          <div class="d-emoji-picker__tab">
+            <button type="button" aria-label="thumbs up"><img class="d-icon d-icon--size-500" alt="thumbs up" aria-label="thumbs up" title="thumbs up" src="https://static.dialpadcdn.com/joypixels/png/unicode/32/1f44d.png" /></button>
+            <button type="button" aria-label="thumbs up: medium-light skin tone">
+              <img
+                class="d-icon d-icon--size-500"
+                alt="thumbs up: medium-light skin tone"
+                aria-label="thumbs up: medium-light skin tone"
+                title="thumbs up: medium-light skin tone"
+                src="https://static.dialpadcdn.com/joypixels/png/unicode/32/1f44d-1f3fc.png"
+              />
+            </button>
+            ...
+          </div>
+        </div>
+        <div class="d-emoji-picker__alignment" data-index="2">
+          <p>Smileys and people</p>
+          ...
+        </div>
+        <div class="d-emoji-picker__alignment" data-index="3">
+          <p>Nature</p>
+          ...
+        </div>
+        ...
+      </div>
+    </div>
+  </div>
+  <div class="d-emoji-picker--footer">
+    <div class="d-emoji-picker__data"></div>
+    <div data-qa="skin-selector">
+      <div class="d-emoji-picker__skin-list" style="display: none;">
+        <button class="">
+          <img class="d-icon d-icon--size-500" alt=":wave_tone1:" aria-label=":wave_tone1:" title=":wave_tone1:" src="https://static.dialpadcdn.com/joypixels/png/unicode/32/1f44b-1f3fb.png" />
+        </button>
+        ...
+      </div>
+      <div class="d-emoji-picker__skin-selected">
+        <div data-qa="dt-tooltip-container">
+          <span data-qa="dt-tooltip-anchor">
+            <button aria-label="Change default skin tone" tabindex="-1">
+              <img class="d-icon d-icon--size-500" alt=":wave:" aria-label=":wave:" title=":wave:" src="https://static.dialpadcdn.com/joypixels/png/unicode/32/1f44b.png" />
+            </button>
+          </span>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+'
+vueCode='
+<dt-emoji-picker
     :skin-tone="Default"
-    :skin-selector-button-tooltip-label="'Change default skin tone'"
+    skin-selector-button-tooltip-label="Change default skin tone"
     :tab-set-labels="tabSetLabels[]"
     :recently-used-emojis="recentlyUsedEmojis[]"
```

Comment:
These two are a bit confusing because it's not what you would actually pass to this component. Maybe show a couple array entries and ellipses the rest. Ex:

```
:tab-set-labels=['Most recently used', 'Smileys and people', ...]
```